cves/CVE-2018-1305.yml

```diff
@@ -74,8 +79,8 @@ fixes:
    - commit:
      note:
 vccs:
-  - commit:
-    note:
+  - commit: ece65c1a428094b1c6c17de3d7593f64e1bb1286
```

Comment:
There were quite a couple fixes. Did these fixes all involve a different step in properly fixing the bug? Or were they attempts and fails until the final one solved the problem?

Reply:
It seems like the issue was farther reaching than just one file and that is why there was so much changed in the commit